### DISCUSSION TITLE
Any chance of signing up to Hacktoberfest?

### DISCUSSION THREAD

**Discussion by GrayedFox** [eslint/eslint#13743] [General] [answered]
Hi there! I was wondering if the ESLint core contributors might consider adding the hacktoberfest label and opting into hacktoberfest? Could be a great way to encourage newcomers (or oldcomers/lazy bums like myself who let a few things go stale!).

https://hacktoberfest.digitalocean.com/hacktoberfest-update

Keep up the good work 👍🏾

**Comment by nzakas** [comment]
We haven't had a great experience so far with Hacktoberfest so far (lots of useless PRs), so I'm not sure how enthusiastic we are about opting-in. I'll discuss with the team.

**Reply by GrayedFox** [reply]
Fair enough, thanks for considering it 👍🏾

**Comment by nzakas** [comment, marked as answer]
We discussed at the last TSC meeting and decided against participating this year.
